### PR TITLE
Sensor graph on KATGUI (karoo) includes a hyperlink to kat-flap sensor graph (mkat and rts)

### PR DESCRIPTION
**Pull request by tockards**
This PR adds functionality to katgui that displays a link that is stored in config. 
The link is displayed as a database icon in the `Tools` toolbar of sensorgraph 
go_to_flap01
When hovered
go_to_flap02

The destination of the link is set and retrieved from the `[other]` config item in base_conf.conf and the related PR can be found at https://github.com/ska-sa/katcamconfig/pull/651

### CODE REVIEW

File: app/sensor-graph/sensor-graph.js [hunk 1]
```diff
@@ -84,6 +84,21 @@
           }
         ];
 
+        vm.getNewKatstoreLink = function () {
+	    ConfigService.getSystemConfig()
+                .then(function(systemConfig){
+                    if (systemConfig.other.katstore){
+                        vm.new_katstore_url = systemConfig.other.katstore;
```
Comment (ajoubertza) [inline app/sensor-graph/sensor-graph.js]: The katcamconfig PR has the key `flap_sensorgraph` - which one is correct?

File: app/sensor-graph/sensor-graph.js [hunk 2]
```diff
@@ -84,6 +84,21 @@
...
+                    }
+		    else{
+		        vm.new_katstore_url =  false;
+		    }
+		 }, function(error){
+                        $log.error(error);
+		    });
+	        return vm.new_katstore_url;
+         };
+	vm.getNewKatstoreLink();
```
Comment (ajoubertza) [inline app/sensor-graph/sensor-graph.js]: Indentation here looks inconsistent with the rest of the code.

File: app/sensor-graph/sensor-graph.html [hunk 3]
```diff
@@ -10,6 +10,7 @@
         </md-toolbar>
         <div flex class="md-whiteframe-z1" layout="column" layout-align="start center" md-theme="{{themePrimaryButtons}}">
 
+            <a ng-if="vm.new_katstore_url" ng-href="{{vm.new_katstore_url}}">Link To New Katstore Sensorgraph</a>
```
Comment (ajoubertza) [inline app/sensor-graph/sensor-graph.html]: Screenshot tool tip is "Go to Flap Sensorgraph", which differs to this - which one is correct?

Comment (tockards) [reply app/sensor-graph/sensor-graph.html]: @ajoubertza I pushed the code correct, the code in the PR was behind.

**Pull request merged**